docs/install/requirements.md

```diff
@@ -46,8 +48,26 @@ InvenioRDM depends on the following services. During the installation we start t
 
 ### Hardware
 
-We usually run InvenioRDM on machines that have at least 8GB of RAM and at least
-4 cores.
+We usually run InvenioRDM on machines that have at least 8GB of RAM and at least 4 cores.
+
+### Python virtual environments
+
+Because we want to avoid cluttering the Python packages of the system or user with InvenioRDM dependencies, `invenio-cli` uses virtual environments.
+This is done by interacting with `pipenv` behind the scenes, which is listed as a requirement and can simply be installed via `pip`.
+For enabling the use of a different Python version in the virtual environment than the one installed globally, a Python version manager such as `pyenv` (or `asdf` with `asdf-python`) is required.
+
+For simplicity, we recommend to go with `pyenv` here.
+In some cases, it can be installed via the system's package manager (e.g. `pacman -S pyenv`).
+Otherwise, you can find the installation instructions on the [project's GitHub page](https://github.com/pyenv/pyenv/#installation) or use their [automatic installer](https://github.com/pyenv/pyenv-installer) (note the required [dependencies for locally building Python](https://github.com/pyenv/pyenv/wiki#suggested-build-environment)).
+
+To make sure that `pyenv` is initialized when your shell starts, you have to add a few lines to your shell configuration file (e.g. `~/.zshrc`, `~/.bashrc`, ...):
+
+```bash
+export PYENV_ROOT="$HOME/.pyenv/bin"
+export PATH="$PATH:$PYENV_ROOT"
+eval "$(pyenv init -)"
+eval "$(pyenv virtualenv-init -)"
+```
```

Comment:
I wouldn't include that and just leave it at: "check the pyenv repository". These things change with time and with shells (we don't want to keep-up with their instructions).